fhir-server-test/src/test/java/com/ibm/fhir/server/test/operation/EraseOperationTest.java

```diff
@@ -194,8 +194,15 @@ private void eraseResource(String resourceType, String logicalId, boolean error,
      * @param reason should include the reason?
      * @param reasonMsg the string to provide
      */
-    private void eraseResource(String resourceType, String logicalId, boolean error, String msg, boolean patient, boolean reason, String reasonMsg) {
-        Entity<Parameters> entity = Entity.entity(generateParameters(patient, reason, reasonMsg, Optional.empty()), FHIRMediaType.APPLICATION_FHIR_JSON);
+    private void eraseResource(String resourceType, String logicalId, boolean error, String msg,
+            boolean patient, boolean reason, String reasonMsg) {
+        Parameters requestBody = generateParameters(patient, reason, reasonMsg, Optional.empty());
+        if (DEBUG) {
+            System.out.println("Invoking $erase on " + resourceType + "/" + logicalId +
```

Comment:
single quotes around the values.

Reply:
its just test output, but i went ahead and added single quotes around resourceType/logicalId (but not around the request body)